docs/iot/debugging.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Debug .NET apps on Raspberry Pi 
+description: Learn how to debug .NET apps on Raspberry Pi and similar devices.
+author: camsoper
+ms.author: casoper
+ms.date: 11/2/2020
+ms.topic: overview
+ms.prod: dotnet
+---
+
+# Debug .NET apps on Raspberry Pi
+
+Debugging .NET apps running on ARM-based IoT devices like Raspberry Pi presents a unique challenge. It is possible to develop .NET apps on ARM devices. However, OmniSharp, which is required for debugging .NET apps outside of Visual Studio, is not compatible with ARM devices. As a result, apps must be debugged remotely from a compatible platform.
+
+## Debug from Visual Studio on Windows
```

Comment:
I recommend using either zone pivots or tabs for this doc. The reader should be able to select from either VS or VS Code.

Reply:
Point taken. I'd say the pivots should be .NET CLI or Visual Studio though.